pkg/frontend/assets/flowhouse.js

```diff
@@ -142,27 +142,58 @@ function drawChart() {
 }
 
 function renderChart(rdata) {
-  pres = Papa.parse(rdata.trim())
+  pres = Papa.parse(rdata.trim());
+
+  var filtered = [pres.data[0]];
```

Comment:
[sand] this is a little convoluted, i'd prefer using `Array.slice` and `const/let`

```suggestion
  const filtered = pres.data.slice(1);
```

can you explain why the first series is always a member of the filter result?

Reply:
that's supposed to be a timestamp element of a flow